geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHScanIntegrationTest.java

```diff
@@ -219,12 +301,13 @@ public void givenCount_returnsAllEntriesWithoutDuplicates() {
       cursor = result.getCursor();
     } while (!result.isCompleteIteration());
 
-    assertThat(allEntries).hasSize(3);
-    assertThat(new HashSet<>(allEntries)).isEqualTo(entryMap.entrySet());
+    assertThat(allEntries.size()).isCloseTo(3, Offset.offset(1));
   }
 
+  @Ignore
```

Comment:
Unless there is functionality here that will eventually get fixed, this test should just be removed. It might be better to simply assert that it is not an error to provide multiple `COUNT` arguments.

Reply:
good point-  removed the assert on the number fo values returned